### DISCUSSION TITLE
I have an issue in validation in nextjs

### DISCUSSION THREAD

**Discussion by yanalshoubaki** [react-hook-form/react-hook-form#10281] [General]
I use useForm with nextjs to handle comment form, i add validation to it, when i make validation error case like this
![image](https://user-images.githubusercontent.com/13173362/232280706-ceda7691-f1e3-4e4e-9018-7f71d116b9fe.png)

after i go to another page, the validation still there, here is a video for issue 
[Screencast from 16 نيسان, 2023 +03 10:34:45 ص.webm](https://user-images.githubusercontent.com/13173362/232280774-1fbaaf74-33ad-4179-bfdc-39c156fbb676.webm)

**Comment by leapful** [comment]
@yanalshoubaki 

Please provide a minimal re-producible codesandbox url that can simulate your form implementation in order to get more investigation.

**Reply by yanalshoubaki** [reply]
I will try to give an example, but this case just on this website, and the project is confidential.

If you have any ideas please share it with me.

**Reply by leapful** [reply]
@yanalshoubaki 

The root cause can be anything on your implementation, for example: FormContext is being used for the whole layout of Next.js page that will keep form state even when navigating to other page.

Without a minimal code sample, it's difficult to know the reason.

**Comment by yanalshoubaki** [comment]
I will close the issue because i can't add an example for the code, thank you @leapful  for your replay
